website/src/components/dynamic-form/dynamic-form.tsx

```
@@ -277,6 +277,11 @@ const GenericFormField = ({
 					/>
 				);
 			case 'ZodEnum':
+				const options = Object.entries(getEnumValues(option, parentOption));
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)